src/Datasource/EntityTrait.php

```diff
@@ -297,9 +297,7 @@ public function &get(string $field): mixed
 
         $method = static::_accessor($field, 'get');
         if ($method) {
-            $result = $this->{$method}($value);
-
-            return $result;
+            return $this->{$method}($value);
```

Comment:
That return by reference doesn't make sense here does it.

Reply:
Should we revert this?